fireapp/pubspec.lock

```diff
@@ -509,6 +509,30 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "6.7.1"
+  leak_tracker:
```

Comment:
Why all these new dependencies? This does not seem like the correct place to introduce them.

Reply:
I am not sure, because this morning I first create the branch wrong and I delete all the things one time, when I do second time I forgot open the docker, is it affect the depedencies?

Reply:
Docker shouldn't affect the dependencies. Please revert these changes

Reply:
I try to revert everything, but I don't know is it successful, sorry for messed up, don't know why this happen, I am still try, I am so sorry for this.